qa/integration-tests/src/test/java/io/camunda/it/client/IncidentQueryTest.java

```diff
@@ -406,20 +410,30 @@ void shouldSortByState() {
         .containsExactlyElementsOf(sortedDesc);
   }
 
+  // TO-DO: Verify the following test case: the getJobKey is returning null, which is not really
```

Comment:
@ralfpuchert  is there any reason why job key comes null here? 
Before the integration test was working because was only one element, but after I add 2 new incidents, as the job key was null, it was throwing error here, so I adjust this test in order to have null check.

Reply:
For visibility:

Update: Me and @ralfpuchert  discussed in a call, we can leave this test as is for now, it could be related (or not) to a delay on import. Ralf will check it separately, and it will be worked in a different PR/issue.